aws_lambda_builders/utils.py

```diff
@@ -222,3 +225,53 @@ def extract_tarfile(tarfile_path: Union[str, os.PathLike], unpack_dir: Union[str
                 raise tarfile.ExtractError("Attempted Path Traversal in Tar File")
 
         tar.extractall(unpack_dir)
+
+
+def glob_copy(source: Union[str, list], destination: str) -> None:
+    items = source if isinstance(source, list) else [source]
+    dest_path = Path(destination)
+    known_paths = []
+    for item in items:
+        if os.path.isabs(item.replace('\\', '/')):
+            raise FileOperationError(operation_name='glob_copy', reason='"{item}" is not a relative path'.format(item=item))
+        files = glob(item, recursive=True)
+        if len(files) == 0:
+            raise FileOperationError(operation_name='glob_copy', reason='"{item}" not found'.format(item=item))
+        for file in files:
+            save_to = str(dest_path.joinpath(file))
+            LOG.debug("Copying resource file from source (%s) to destination (%s)", file, save_to)
+            save_to_dir = os.path.dirname(save_to)
+            if save_to_dir not in known_paths:
+                os.makedirs(save_to_dir, exist_ok=True)
+                known_paths.append(save_to_dir)
+            shutil.copyfile(file, save_to)
+
+
+def robust_rmtree(path, timeout=1):
```

Comment:
If this is only used in tests, we should move it there.

Reply:
Moved to test/functional/test_utils.py